### PR TITLE
Add license key in __openerp__.py

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

**Comment by lmignon** [comment]
:+1:

File: __unported__/account_financial_report/__openerp__.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "version": "2.0",
     "author": "Vauxoo",
     "website": "http://www.vauxoo.com",
+    "license": "AGPL-3",
```
Comment (yvaucher) [inline __unported__/account_financial_report/__openerp__.py]: GPL-3 or later

**Comment by pedrobaeza** [comment]
You need to rebase as I merge one migration

File: __unported__/account_financial_report/__openerp__.py [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "version": "2.0",
     "author": "Vauxoo",
     "website": "http://www.vauxoo.com",
+    "license": "GPL-3 or any later version",
```
Comment (max3903) [inline __unported__/account_financial_report/__openerp__.py:31]: Why GPL for this one ?

Comment (yvaucher) [inline __unported__/account_financial_report/__openerp__.py:31]: Value taken from header

**Comment by yvaucher** [comment]
@pedrobaeza This was rebased

**Pull request merged**